docs/source/_src_user_guide/state_propagation/propagation_setup/console_output.rst

```diff
@@ -0,0 +1,185 @@
+.. _printing_processing_results:
+
+###################################
+Printing and processing the results
+###################################
+
+The results of the numerical propagation are stored in the :ref:`propagation results <propagation_results>`, which can be used to perform further
+analysis of the simulation, and the details of the propagation process. In addition, there are a number of processing and
+output steps that Tudat can perform before, during and after the propagation. Although such settings can be provided directly
+when creating propagator settings (for instance, when calling the :func:`~tudatpy.numerical_simulation.propagation_setup.propagator.translational`
+function), it is typically more convenient to modify these settings afterwards:
+
+.. code-block:: python
+
+    propagator_settings = propagator.translational( ... )
+    console_print_settings = propagator_settings.print_settings
+    post_processing_settings = propagator_settings.processing_settings
+
+where, the resulting settings object for console printing and post-processing are of type
+:class:`~tudatpy.numerical_simulation.propagation_setup.PropagationPrintSettings` and
+:class:`~tudatpy.numerical_simulation.propagation_setup.PropagatorProcessingSettings`
+(with derived class :class:`~tudatpy.numerical_simulation.propagation_setup.SingleArcsPropagatorProcessingSettings`
+for single-arc propagation), respectively. Below, the options available in both these settings objects are described in more
+detail.
+
+.. _auto_processing:
+
+Automatic processing
+====================
+
+Retrieving the results of the numerical propagation is discussed in detail :ref:`here <propagation_results>`.
+In addition to returning data for further analysis by the user, Tudat can be set up to automatically use
+the results of the numerical propagation to update the environment upon the successful completion of the propagation.
```

Comment:
The word 'update' is used here, whereas later the word 'reset' is often used. Should only one word be used here, or is there a distinction that I am not grasping?

Reply:
None whatsoever, thanks for pointing this out. I've changed everything to reset